internal/controller/isbservicerollout/isbservicerollout_controller.go

```diff
@@ -149,18 +149,14 @@ func (r *ISBServiceRolloutReconciler) Reconcile(ctx context.Context, req ctrl.Re
 
 	// Update the Spec if needed
```

Comment:
btw, can you fix the comment I wrote in each of these files which says "Update the Spec if needed" to something like "Update the resource definition (everything except Status)" - my comment wasn't right